src/scripts/main.js

```diff
@@ -1,3 +1,56 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+const message = document.createElement('div');
+
+message.classList.add('message');
+
+const badResult = function (content) {
+  message.classList.add('error-message');
+
+  message.textContent = content;
+  body.append(message);
+
+  return message;
+};
+
+const goodResult = function (content) {
+  message.classList.toggle('error-message', false);
+  message.textContent = content;
+  body.append(message);
+
+  return message;
+};
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.onclick = (e) => {
```

Comment:
in js, we usually use the addEventListener instead of the onClick

Reply:
Still not fixed 
<img width="908" alt="image" src="https://github.com/user-attachments/assets/83a4859b-fde2-4b9a-9605-0205762064b1">